src/Result/LightningInvoice.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Result;
+
+use BTCPayServer\Util\PreciseNumber;
+
+class LightningInvoice extends AbstractResult
+{
+    public function getId(): ?string
```

Comment:
Not sure that an ID could be NULL in the response?

Reply:
It is while we greenfield is bugged out :(

Reply:
Sorry just realizing now that this is invoice not payment, my bad. Will remove.